Source/diablo.cpp

```diff
@@ -289,13 +289,19 @@ void diablo_splash()
 	UiTitleDialog();
 }
 
+void diablo_terminate()
+{
+	init_cleanup();
+	SaveGamma();
+	SDL_Quit();

```

Comment:
The current placement of `SDL_Quit` makes more sens as it is only added if needed (after `SDL_Init`)

Reply:
[https://wiki.libsdl.org/SDL_Quit](url)
'It is safe to call this function even in the case of errors in initialization.'
I think and if SDL not initialized.

Reply:
Sure, but there is no point in changing the current code.